client/my-sites/current-site/index.jsx

```diff
@@ -149,12 +149,19 @@ const CurrentSite = React.createClass( {
 					</span>
 				}
 				{ selectedSite
-					? <Site
-						site={ selectedSite }
-						homeLink={ true }
-						externalLink={ true }
-						onSelect={ this.previewSite }
-						tipTarget="site-card-preview" />
+					? <div>
+						<Site site={ selectedSite } />
+						<a
+							href={ selectedSite.URL }
+							onClick={ this.previewSite }
+							className={ `current-site__view-site${ this.props.isPreviewShowing ? ' selected' : '' }` }
+						>
+							<Gridicon icon="computer" />
+							<span className="current-site__view-site-text">
+								{ translate( 'Preview Site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Site Preview' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).